addon/templates/components/frost-button.hbs

```diff
@@ -1,3 +1,4 @@
+{{! Template for the frost-button component }}
```

Comment:
So are you saying leave it in if it's in a pod, b/c the file name is the same, but delete it if the file name is unique? I can live with that, I guess. I just like having header comments on files as it gives me a place to explain what's going on. Sometimes it's handy to be reminded of what the `tagName` is when looking at the template (info you don't have in that context). I guess that's more of a case-by case decision.

Are you suggesting we not allow people to have header comments on templates, or just suggesting that having them with no more information than the name of the component is pointless?